addon/adapters/rest.js

```diff
@@ -881,11 +881,20 @@ export default Adapter.extend(BuildURLMixin, {
         Ember.run.join(null, reject, error);
       };
 
-      Ember.$.ajax(hash);
+      adapter.ajaxRequest(hash);
     }, 'DS: RESTAdapter#ajax ' + type + ' to ' + url);
   },
 
   /**
+    @method ajaxRequest
+    @private
+    @param {Object} options jQuery ajax options to be used for the ajax request
+  */
+  ajaxRequest(options) {
```

Comment:
I'd prefer to see this named `_ajaxRequest` to further discourage non fastboot consumers.